src/main/webapp/dataset.xhtml

```diff
@@ -143,17 +139,29 @@
                                                 </button>
                                                 <ul class="dropdown-menu pull-right text-left">
                                                     
-                                                    <ui:remove>
                                                     <!-- DOWNLOAD -->
-                                                    <!-- TO-DO #3513 DOWNLOAD ALL LINK -->
                                                     <ui:fragment rendered="#{DatasetPage.canDownloadFiles()}">
                                                         <li class="dropdown-header">#{bundle['dataset.accessBtn.header.download']} <span class="glyphicon glyphicon-download-alt"/></li>
-                                                        <li class="disabled">
-                                                            <span class="ui-commandlink ui-widget ui-state-disabled btn-download">Download (Placeholder)</span>
+                                                        <!-- NORMAL DOWNLOAD BUTTON (NO TABULAR FILES) -->
+                                                        <li jsf:rendered="#{!DatasetPage.isHasTabular()}">
+                                                            <p:commandLink update="@form" actionListener="#{DatasetPage.validateAllFilesForDownloadArchival()}">
+                                                                #{bundle.download} (ZIP, #{DatasetPage.sizeOfDataset})
+                                                            </p:commandLink>
+                                                        </li>
+                                                        <!-- DOWNLOAD ORIGINAL BUTTON (TABULAR FILES PRESENT) -->
+                                                        <li jsf:rendered="#{DatasetPage.isHasTabular()}">
+                                                            <p:commandLink update="@form" actionListener="#{DatasetPage.validateAllFilesForDownloadOriginal()}">
+                                                                #{bundle.downloadOriginal} (ZIP, #{DatasetPage.sizeOfDataset})
+                                                            </p:commandLink>
+                                                        </li>
+                                                        <!-- DOWNLOAD ARCHIVAL FILES (TABULAR FILES PRESENT) -->
+                                                        <li jsf:rendered="#{DatasetPage.isHasTabular()}">
+                                                            <p:commandLink update="@form" actionListener="#{DatasetPage.validateAllFilesForDownloadArchival()}">
```

Comment:
"ZIP" file type labels should be moved to the bundle.

Reply:
There is already a `application/zip=ZIP Archive` entry in the MimeTypeDisplay.properties, not sure if it would be possible to pull from that file, or if we need a new entry in Bundle.properties.

Reply:
Fixed in 140ba41e4.